Makefile

```diff
@@ -46,9 +46,13 @@ ifndef OFFICIAL_BUILD
 endif
 
 build: pre-build
+ifdef IS_WIN
+PLATFORM_SPECIFIC_LDFLAGS := -H windowsgui
+endif
```

Comment:
`IS_WIN` is intended for shell behaviours, not for detecting the windows platform. I had to manually include the windows flags in my local version because it didn't detect windows from my environment (which uses git bash).

Reply:
Fixed by adding a new variable for detecting the platform